besu/src/main/java/org/hyperledger/besu/cli/options/unstable/EvmOptions.java

```diff
@@ -44,9 +46,18 @@ public static EvmOptions create() {
   private Long jumpDestCacheWeightKilobytes =
       32_000L; // 10k contracts, (25k max contract size / 8 bit) + 32byte hash
 
+  @SuppressWarnings({"FieldCanBeFinal", "FieldMayBeFinal"})
+  @CommandLine.Option(
+      names = {KZG_TRUSTED_SETUP_FILE_PATH},
+      description =
+          "absolute file path on the host system which points to the trusted KZG setup file to use",
+      arity = "1")
+  private String kzgTrustedSetupFilePath = null;
```

Comment:
Could this be `Path` instead of `String` like is done for other similar options that point to files? this will also help in fail fast in case the argument does not point to an existing file

Reply:
bump